components/board/server_test.go

```diff
@@ -786,3 +790,172 @@ func TestServerGetDigitalInterruptValue(t *testing.T) {
 		})
 	}
 }
+
+type streamTicksServer struct {
```

Comment:
Does this test file still have remnants of when you copied over the backend of the gen files?

Reply:
No, the server side stream ticks function (called on line 934) takes a server as a parameter so this is just the server for tests.

Reply:
Cool I'll look throught it more closely, saw a bunch of pb and wondered.

Reply:
nice test cases actually. and good job figuring out that race